ui/perfherder/compare/CompareTableView.jsx

```diff
@@ -274,6 +361,20 @@ export default class CompareTableView extends React.Component {
           message={genericErrorMessage}
         >
           <React.Fragment>
+            <Row className="justify-content-center">
+              <Alert color="info">
+                A{' '}
+                <a
+                  href={perfCompareURL}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  PerfCompare
+                </a>{' '}
+                comparison is now available.
```

Comment:
What do you think about:
`Try out the same comparison <a>with our new PerfCompare tool</a>!`